db/schema.rb

```diff
@@ -22,12 +22,42 @@
     t.text     "summary"
     t.integer  "parent_id"
     t.integer  "children_count"
-    t.datetime "created_at",                     null: false
-    t.datetime "updated_at",                     null: false
-    t.boolean  "placeholder",    default: false, null: false
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
     t.index ["slug"], name: "index_categories_on_slug", using: :btree
   end
 
+  create_table "content_blocks", force: :cascade do |t|
```

Comment:
We need to remove this from schema.rb - it is an artefact from a production database dump and isn't actually a part of our application anymore.

When loading production data we need to take care to locally drop this table to avoid this.